cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -33,7 +33,7 @@ import (
 	"github.com/azure/azure-dev/cli/azd/pkg/convert"
 	"github.com/azure/azure-dev/cli/azd/pkg/environment"
 	"github.com/azure/azure-dev/cli/azd/pkg/infra"
-	. "github.com/azure/azure-dev/cli/azd/pkg/infra/provisioning"
+	"github.com/azure/azure-dev/cli/azd/pkg/infra/provisioning"
```

Comment:
IMO, there should be just one package for the provisioning which includes the core components and all the providers.

For `azd pipeline config` I used the `provisioning package model` as the base start, but decided not to create a new package for each provider.

Looking at packaged from go, like the crypto package, it seems like the directories approach is the idiomatic way. Removing the use of dot would put us in that place, which looks like the right thing